lib/node_modules/@stdlib/math/base/special/pow/lib/main.js

```diff
@@ -178,11 +178,11 @@ var LOG_WORKSPACE = [ 0.0, 0.0 ]; // WARNING: not thread safe
 *
 * @example
 * var v = pow( 3.141592653589793, 5.0 );
-* // returns ~306.0197
+* // returns 306.0196847852814
```

Comment:
Similarly, would be interested in seeing the lint error for this and L185.

Reply:
Now that I'm reverting this change, the lint error doesn't appear 😕